src/fzx/tui/term_app.cpp

```diff
@@ -147,24 +148,28 @@ void TermApp::redraw()
           ++p;
           if (!highlighted) {
             highlighted = true;
-            mTTY.put("\x1B[33m"sv);
+            mTTY.setFg(mPalette.mMatchFg);
           }
         } else {
           if (highlighted) {
             highlighted = false;
-            mTTY.put("\x1B[0m"sv);
+            mTTY.setFg(TrueColor("#ffffff"));
```

Comment:
Should revert to default color here, kinda like this
```suggestion
            if (iscursor) {
              mTTY.setFg(TrueColor("#ffffff"));
            } else {
              mTTY.put("\x1B[39m"sv);
            }
```
except maybe add the default color to `TermColor`

Reply:
Also `mPalette.mCursorFg` instead of hardcoded `#ffffff`